<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RightClickMenuDialog</class>
 <widget class="QDialog" name="RightClickMenuDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>116</width>
    <height>176</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="transferBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>35</y>
     <width>116</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>发起转账</string>
   </property>
  </widget>
  <widget class="QPushButton" name="renameBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>116</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>修改账户名</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exportBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>105</y>
     <width>116</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>导出私钥</string>
   </property>
  </widget>
  <widget class="QPushButton" name="copyBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>116</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>复制地址</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>116</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>删除账户</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
